src/lab11.py

```diff
@@ -45,28 +46,28 @@ def get_font(size, weight, style):
         FONTS[key] = font
     return skia.Font(FONTS[key], size)
 
+DEFAULT_STYLE_SHEET = CSSParser(open("browser11.css").read()).parse()
+
 def parse_color(color):
     if color == "white":
         return skia.ColorWHITE
-    elif color == "lightblue":
-        return skia.ColorSetARGB(0xFF, 0xAD, 0xD8, 0xE6)
-    elif color == "orange":
-        return skia.ColorSetARGB(0xFF, 0xFF, 0xA5, 0x00)
+    elif color == "black":
+        return skia.ColorBLACK
     elif color == "red":
         return skia.ColorRED
-    elif color == "green":
-        return skia.ColorGREEN
-    elif color == "blue":
-        return skia.ColorBLUE
-    elif color == "gray":
-        return skia.ColorGRAY
-    elif color == "lightgreen":
-        return skia.ColorSetARGB(0xFF, 0x90, 0xEE, 0x90)
+    elif color.startswith("#"):
+        r = int(color[1:3], 16)
+        g = int(color[3:5], 16)
+        b = int(color[5:7], 16)
+        a = int(color[7:9], 16) if len(color) == 9 else 255
+        return skia.Color(r, g, b, a)
     else:
-        return skia.ColorBLACK
+        raise Exception("Unknown color " + color)
```

Comment:
Don't do this, or the browser will crash on our home page.

Reply:
Oh lol right.